### PR TITLE
Bumps in garden : ign-rendering7

### PR DESCRIPTION
**Pull request by methylDragon**
See https://github.com/ignition-tooling/release-tools/issues/685

### CODE REVIEW

**Review by chapulina — APPROVED** [review]
Just minor comments, LGTM once CI is 🟢

File: test/integration/render_pass.cc [hunk 1]
```diff
@@ -385,7 +385,7 @@ void RenderPassTest::DepthGaussianNoise(const std::string &_renderEngine)
       // Note: internal texture format used is RGB with no alpha channel
       // We observed the values can be either 255 or 0 but graphics card
       // drivers are free to fill it with any value they want.
-      // This should be fixed in ogre 2.2 in ign-rendering6 which forbids
+      // This should be fixed in ogre 2.2 in ign-rendering7 which forbids
```
[the remark this answers is not in the document]
Comment (methylDragon) [reply test/integration/render_pass.cc]: https://github.com/ignitionrobotics/ign-rendering/pull/608/commits/959bcc2220a96d5d07395e2525d3891a3de74bbd

File: test/integration/depth_camera.cc [hunk 2]
```diff
@@ -286,7 +286,7 @@ void DepthCameraTest::DepthCameraBoxes(
       // Note: internal texture format used is RGB with no alpha channel
       // We observed the values can be either 255 or 0 but graphics card
       // drivers are free to fill it with any value they want.
-      // This should be fixed in ogre 2.2 in ign-rendering6 which forbids
+      // This should be fixed in ogre 2.2 in ign-rendering7 which forbids
```
[the remark this answers is not in the document]
Comment (methylDragon) [reply test/integration/depth_camera.cc]: https://github.com/ignitionrobotics/ign-rendering/pull/608/commits/959bcc2220a96d5d07395e2525d3891a3de74bbd

**Comment by chapulina** [comment]
All 🟢 !

**Pull request merged**